File: Repeatingfirst.py
def firstRepeat(arr):
    for i in range(len(arr)):
        for j in range(i+1,len(arr)):
            if arr[i] == arr[j]:
                return arr[i]
    return -1

my_list = [1,2,2,4,6,5,4]
print(firstRepeat(my_list))

File: RepeatingFirst.java
public class RepeatingFirst {
    static int firstRepeat(int arr[]) {
        for (int i = 0; i < arr.length; i++) {
            for (int j = i + 1; j < arr.length; j++) {
                if (arr[i] == arr[j]) {
                    return arr[i];
                }
            }
        }
        return -1;
    }

    public static void main(String[] args) {
        int arr[] = { 1, 2, 5, 6, 2, 6 };
        System.out.println(firstRepeat(arr));
    }
}
